.circleci/config.yml

```diff
@@ -62,6 +67,20 @@ jobs:
           name: Dry run environments playbooks
           command: .circleci/docker-run.sh make test-setup-environments
 
+  check-terraform:
```

Comment:
Could you please use the same keys conventions in the file already with `_` instead of `-`.

Reply:
Also please move all the steps of this job into `build_check` job. Don't forget to run the command with `.circleci/docker-run.sh make your_recipe`